internal/brokers/broker.go

```diff
@@ -125,6 +127,18 @@ func (b Broker) IsAuthorized(ctx context.Context, sessionID, authenticationData
 	return access, userInfo, nil
 }
 
+// AbortSession calls the broker corresponding method, stripping broker ID prefix from sessionID.
+func (b Broker) AbortSession(ctx context.Context, sessionID string) (err error) {
+	sessionID = strings.TrimPrefix(sessionID, fmt.Sprintf("%s-", b.ID))
```

Comment:
Maybe, now that it’s used in many many places, we should just have a small private function doing this?